pkg/dockerclient/shell.go

```diff
@@ -792,7 +798,11 @@ func (c *ShellClient) CreateVolume(options *CreateVolumeOptions) error {
 func (c *ShellClient) DeleteVolume(volumeName string) error {
 	c.logger.DebugWith("Deleting docker volume", "volumeName", volumeName)
 	if !volumeNameRegex.MatchString(volumeName) {
-		return errors.New("Invalid volume name to delete")
+		// if a name doesn't match regexp, volume could not be created,
+		// but we don't want to fail in that case, just do nothing
```

Comment:
why don't we want to fail here?
Is it possible we have a volume with a different name?

Reply:
@TomerShor no, volumes always should match this expression to be created. In fail-flow in past versions we didn't reach a step where we create a volume, so if volume name doesn't match expression,  it doesn't exist and we don't need to fail on deletion it.

Reply:
OK, I changed a couple of words to make it clearer:
```suggestion
		// if a name doesn't match regexp, volume was not created in the first place,
		// so we don't want to fail in that case, just do nothing
```